src/GameDataReader.Tests/BattlefieldVietnam/GameDataReadersTests.cs

```diff
@@ -37,4 +39,35 @@ public void GameDataReaders_GetAccessor_IsNotSingleton()
         bfVietnamDataReader1.GetHashCode().Should()
             .NotBe(bfVietnamDataReader2.GetHashCode());
     }
+
+    [Test]
+    public void GameDataReaders_BfVietnam_IsTrueProfileFileExists()
+    {
+        var GameName = "Battlefield Vietnam";
+        var ModName = "BfVietnam";
+
+        var globalRefractorV1ConfigFile = new GlobalRefractorV1ConfigFile(GameName, ModName);
+        var filePath = globalRefractorV1ConfigFile.GetFilePath();
+
+        Console.WriteLine($"File Path: {filePath}");
+
+        Assert.IsTrue(File.Exists(filePath));
+    }
+
+    [Test]
```

Comment:
Add this, so the test runner on GitHub doesn't fail:
```csharp
[Explicit("Only run this test on a real Windows machine, for end-to-end testing.")]
```

Reply:
Hey @TwitchPlaysBF2

Thank you for the kind words regarding my issue summary and the fixes that I've worked on so far.
I was glad to run across Battlefield Rich Presence, while I was searching for Discord bots that I could add to the MoonGamers Battlefield 1942 Discord server.  When I discovered that Battlefield Rich Presence didn't seem to work with Battlefield 1942, I decided that I wanted to figure out why and see if I could get it working.  I am still kind of a newbie when it comes to programming in C#. :)  Prior to my current job, I had been programming in Borland Delphi 5 (Object Pascal) for 10 years on a legacy Windows XP era program.  Visual Studio and .NET have been a breath of fresh air compared to the obsolete Borland Delphi 5 IDE I had to use.


**Regarding the kernel32 DLL import/GetMainModuleFileName change:**
Now that I take a second look at the code and at what I was trying to accomplish, I agree that the "Process.GetProcessesByName" code above will work correctly for getting the path of the game process executable.

When I was looking for code to get the path of the game process executable, I was trying to take into account that the games could be running under 32bit or 64bit operating systems.
I realize now, that BF1942 and BFV are both 32bit games and the solution below is not necessary and the code can be simplified.
https://stackoverflow.com/questions/5497064/how-to-get-the-full-path-of-running-process/48319879#48319879

Reply:
Delphi wow 😀 well I hope C# is not the Delphi of the future 🤣

Regarding the simplification - alright, sounds good!